math/minuit2/src/HessianGradientCalculator.cxx

```diff
@@ -83,6 +83,8 @@ HessianGradientCalculator::DeltaGradient(const MinimumParameters &par, const Fun
    unsigned int startElementIndex = mpiproc.StartElementIndex();
    unsigned int endElementIndex = mpiproc.EndElementIndex();
 
+   MnFcnCaller fcnCaller{fFcn};
```

Comment:
This will be a performance improvement when using `strategy==2`, where the HessianGradientCalculator is used.